data-reconciliation-app/src/models/summary-record.ts

```diff
@@ -0,0 +1,145 @@
+import { MINIMUM_VOTES_THRESHOLD } from "../utils/constants";
+import { ServiceResult } from "../utils/service-result";
+import { DataAttributeType } from "./data-record";
+import { ReconciledRecord } from "./reconciled-record";
+
+export enum SummaryGroupStatus {
+  NotEnoughVotes = "NOT_ENOUGH_VOTES",
+  LackOfConsensus = "LACK_OF_CONSENSUS",
+  InConsensus = "IN_CONSENSUS",
+}
+
+export enum SummaryStatus {
+  Majority = "Majority",
+  Minority = "Minority",
+}
+
+export interface SummaryRecordProps {
+  key: string;
+  value: DataAttributeType;
+  type: string;
+  status: SummaryStatus;
+  groupStatus: SummaryGroupStatus;
+  membersInAgreementCount: number;
+  membersInDisagreementCount: number;
+  votesCount: number;
+  uniqueValuesCount: number;
+}
+
+export class SummaryRecord implements SummaryRecordProps {
+  readonly key: string;
+  readonly value: DataAttributeType;
+  readonly type: string;
+  readonly status: SummaryStatus;
+  readonly groupStatus: SummaryGroupStatus;
+  readonly membersInAgreementCount: number;
+  readonly membersInDisagreementCount: number;
+  readonly activeMembersCount: number;
+  readonly votesCount: number;
```

Comment:
Is this field the same as totalVoteCount which will be added by a governance code change? 

If so, do we want to remove it and include it as part of the code change?